apollo-ast/src/main/kotlin/com/apollographql/apollo3/ast/api.kt

```diff
@@ -88,12 +87,14 @@ fun GQLDocument.validateAsSchemaAndAddApolloDefinition(): GQLResult<Schema> {
 /**
  * Validates the given document as an executable document.
  *
+ * @param fieldsOnDisjointTypesMustMerge Whether fields with different shape are disallowed to be merged in disjoint types.
```

Comment:
Suggestion to rephrase the comment from negative voice to positive. My brain tends to struggle with negations:

```suggestion
 * @param fieldsOnDisjointTypesMustMerge set to false to relax the standard GraphQL [FieldsInSetCanMerge](https://spec.graphql.org/draft/#FieldsInSetCanMerge()) 
 * and allow fields of different types at the same Json path as long as their parent types are disjoint.
```